docs/tutorials/basics.md

```diff
@@ -0,0 +1,117 @@
+ITowns is a javascript framework for 3D geographic data visualisation. 
+It can display a wide range of data such as ortho-images, Digital Elevation Models (DEM) or 3D models.
+
+ITowns is based on [Three.js](https://threejs.org/), which is a javascript library that implements WebGL to render sophisticated 3D Geometry on a webpage.
+
+In this tutorial, we shall be introduced to the fundamentals of iTowns : how it displays geographic data.
+
+***
+
+## Data visualization with iTowns
+
+The support to display anything within iTowns is called a `View`.
+
+To provide geographic data visualization, a `View` contains a combination of `Layers` which, in a digital map environment, are visual representations of any geographic data.
+In other words, all data that are displayed using iTowns are supported by some `Layer`.
+
+Each `View` in iTowns displays 3D data in a unique Coordinates Reference System (CRS). 
+Yet, the data displayed within a `View` do not necessarily have to be in the `View`'s CRS.
+ITowns comes with two pre-made `View` types : `GlobeView` and `PlanarView`. 
+Each of these allows visualizing data from different types (raster or vector) and from different CRS.
+On this matter, iTowns distinguishes two cases : 
+
+- The first one regards data that are to be displayed as raster. 
+  These data can either be original raster data or vector data whose representation is projected on the ground. 
+  Data from this type are displayed in a `GlobeView` if their source CRS is [WGS 84](https://epsg.io/4326) or [Pseudo-Mercator](https://epsg.io/3857).
+  However, if their source CRS defines a local projection (such as [RGF93 / Lambert 93](https://epsg.io/2154) for instance), they are displayed in a `PlanarView`.
+- On another hand, vector data that are to be displayed as 3D objects can be displayed in any type of `View`, regardless of their source CRS.
+
+It is important to aknowledge the following facts regarding original raster data and `Views` :
+- a `GlobeView` allows displaying multiple raster data from different source CRS,
```

Comment:
GlobeView: 2 crs on raster : WGS84 and PM

Reply:
I modified the line.